src/fileformat/types/certificate_table/certificate.cpp

```diff
@@ -244,7 +244,7 @@ void Certificate::loadSignatureAlgorithm()
 void Certificate::loadSerialNumber()
 {
 	if (auto sn = X509_get_serialNumber(certImpl))
-		retdec::utils::bytesToHexString(sn->data, sn->length, serialNumber);
+		retdec::utils::bytesToHexString(sn->data, sn->length, serialNumber, 0, 0, false);
```

Comment:
Consider changing
```
retdec::utils::bytesToHexString(sn->data, sn->length, serialNumber, 0, 0, false);
```
to
```
retdec::utils::bytesToHexString(sn->data, sn->length, serialNumber, 0, 0, /*uppercase=*/false);
                                                                          ^^^^^^^^^^^^^^
```
so the meaning of the `false` argument is clear. Unfortunately, C++ does not have something akin to keyword arguments from Python :(.